everviz/plugins/_crossplot_plugin/_get_indexed_control.py

```diff
@@ -0,0 +1,15 @@
+def _identify_indexed_controls(controls):
+    indexed_controls = []
+    base_names = []
+    for control in controls:
+        split_string = control.split("-")
+        if len(split_string) == 2:
+            base_name, index = split_string
+            try:
+                int(index)
+                if base_name not in base_names:
+                    base_names.append(base_name)
+                indexed_controls.append(control)
+            except ValueError:
+                continue
+    return indexed_controls, base_names
```

Comment:
`indexed_controls` are returned but they are never used.

Reply:
Removed from this PR, will fix in another PR.